### ISSUE TITLE
asciidoc is end of life, replace with asciidoctor?

### ISSUE THREAD

**Issue by dak180** [fink/fink-distributions#123]
See: https://github.com/asciidoc/asciidoc/releases/tag/8.6.10 , https://github.com/asciidoc/asciidoc/issues/103#issuecomment-322077321 and https://github.com/asciidoc/asciidoc3/pull/1#issuecomment-368262945

**Comment by clepple** [comment]
I have a bit of a backlog at the moment, so if someone else gets to this first, I won't be offended.

As for asciidoctor: I will be happy to work with someone to do Conflicts/Replaces if it makes sense to install to the same script names (asciidoc and a2x). However, I have no experience with Ruby packages.

**Comment by nieder** [comment]
I updated to asciidoc 8.6.10 (and turned on tests). Asciidoctor is apparently a ruby program, so that'll take a bit more work.

**Comment by danielj7** [comment]
I added an asciidoctor package. The script names are different, `asciidoctor` versus `asciidoc` so there's no need for Conflicts/Replaces. I added ruby 2.3, 2.4 and 2.5 versions plus an `asciidoctor` bundle package that can be used if the ruby version doesn't matter.

Packages that need to process asciidoc files should replace `asciidoc document.adoc` with `asciidoctor -a compat-mode document.adoc` to get compatible output.
